nipoppy/data/examples/sample_pipelines/freesurfer_stats_and_qc-0.1.0/script.py

```diff
@@ -379,10 +379,30 @@ def run_single_stats(
             verbose=verbose,
         )
 
+        # FreeSurfer 6 has duplicate columns between files
+        for col in ["BrainSegVolNotVent", "eTIV"]:
+            first = None
+            for df_to_concat in [df_aseg, df_aparc_lh, df_aparc_rh]:
+                if col in df_to_concat:
+                    if first is None:
+                        first = df_to_concat[col]
+                    else:
+                        common_participants = list(
+                            set(first.index).intersection(set(df_to_concat.index))
+                        )
+                        if first.loc[common_participants].equals(
+                            df_to_concat.loc[common_participants, col]
+                        ):
+                            df_to_concat.drop(col, axis="columns", inplace=True)
+                            print(f"Dropped duplicate column {col} from aparc file.")
```

Comment:
Why would there be duplicates?

Reply:
When I ran this on FreeSurfer 6 I got duplicate columns (with identical contents) in the left/right aparc files. Like `eTIV`, `BrainSegVolNotVentSurf`, and `SupraTentorialVolNotVentVox`